Project.toml

```diff
@@ -19,6 +21,7 @@ MultivariateStats = "6f286f6a-111f-5878-ab1e-185364afe411"
 PackageExtensionCompat = "65ce6f38-6b18-4e1d-a461-8949797d7930"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Intentional?

Reply:
good catch, we don't want to add this as a dep. @JorgeLuizFranco if you have Revise in your global Julia environment, you shouldn't have to add to the project.